src/routes/notifications/v1/notifications-v2compatible.controller.spec.ts

```diff
@@ -189,6 +194,76 @@ describe('Notifications Controller (Unit)', () => {
       },
     );
 
+    it(`Should throw if the timestamp in request is older than ${REGISTRATION_TIMESTAMP_EXPIRY_MINUTES} minutes ago`, async () => {
+      loggingService.error = jest.fn();
+      const registerDeviceDto = await buildInputDto();
+      const now = new Date();
+      const expiryThresholdSeconds = REGISTRATION_TIMESTAMP_EXPIRY + 60;
+      const expiredDateTime = new Date(
+        now.getTime() - expiryThresholdSeconds * 1000,
+      );
+      const expiredTimestampInSeconds = Math.floor(
+        expiredDateTime.getTime() / 1000,
+      );
+      registerDeviceDto.timestamp = expiredTimestampInSeconds.toString();
+      networkService.get.mockImplementation(({ url }) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
+          ? Promise.resolve({
+              data: rawify(chainBuilder().build()),
+              status: 200,
+            })
+          : rejectForUrl(url),
+      );
+      networkService.post.mockImplementation(({ url }) =>
+        url.includes('/api/v1/notifications/devices/')
+          ? Promise.resolve({ data: rawify({}), status: 200 })
+          : rejectForUrl(url),
+      );
+
+      await request(app.getHttpServer())
+        .post('/v1/register/notifications')
+        .send(registerDeviceDto)
+        .expect(400)
+        .expect({
+          message: 'The signature is expired!',
+          error: 'Bad Request',
+          statusCode: 400,
+        });
+
+      expect(notificationServiceV2.upsertSubscriptions).not.toHaveBeenCalled();
+    });
+
+    it(`Should pass if the timestamp in request is newer than ${REGISTRATION_TIMESTAMP_EXPIRY_MINUTES} minutes ago`, async () => {
+      loggingService.error = jest.fn();
+      const registerDeviceDto = await buildInputDto();
+      const now = new Date();
+      const expiredDateTime = new Date(now.getTime() * 1000);
```

Comment:
Nit: similar to above, we should reference `registerDeviceDto.timestamp`.

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2192/commits/d0a24a9a6bfca0ad2191fa61eebd8737ac1d6b69